about.html

```diff
@@ -0,0 +1,116 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="src/assets/stylesheets/style.css">
+  <script src="https://kit.fontawesome.com/e2597287a1.js" crossorigin="anonymous"></script>
+  <title>Minute Clinic Conference</title>
+</head>
+<body>
+  <div class="app-cont">
+    <header>
+      <div class="about-cont">
+        <div class="sub-header">
+          <div class="sub-header-flex">
+            <ul>
+              <li class="sub-header-icon"><i class="fa-brands fa-facebook-f"></i></li>
+              <li class="sub-header-icon"><i class="fa-brands fa-twitter"></i></li>
+              <li>English</li>
+              <li>My Page</li>
+              <li>Logout</li>
+            </ul>
+          </div>
+        </div>
+        <div class="header-cont">
+          <nav>
+            <div class="header-links">
+              <div class="header-links-position">
+                <a href="index.html" class="home-link"><img src="src/assets/images/pngfind.com-cvs-logo-png-1550097.png" alt=""></a>
+                <ul>
+                  <li><a class="nav-link" href="index.html">Home</a></li>
+                  <li><a class="nav-link active" href="about.html">About</a></li>
+                  <li><a class="nav-link" href="">Program</a></li>
+                  <li><a class="nav-link" href="">Join</a></li>
+                  <li><a class="nav-link" href="">Sponsor</a></li>
+                  <li><a class="nav-link" href="">News</a></li>
+                  <li><a href="index.html"><button class="nav-btn">Minute Clinic</button></a></li>
```

Comment:
- [ ] For a better user experience on your page, Kindly make sure when the user clicks on a link they may be redirected to that specific section on the page :smile: 
`eg: Program, Sponsor, Speakers ...`